packages/material-ui/src/styles/experimentalStyled.test.js

```diff
@@ -178,5 +177,19 @@ describe('experimentalStyled', () => {
         height: '400px',
       });
     });
+
+    it('should resolve the sx prop', () => {
+      render(
+        <ThemeProvider theme={theme}>
+          <Test data-testid="component" sx={{ color: 'primary.main' }}>
+            Test
+          </Test>
+        </ThemeProvider>,
+      );
+
+      expect(screen.getByTestId('component')).toHaveComputedStyle({
```

Comment:
Sometime, I find this approach simpler
```suggestion
      expect(container.firstChild).toHaveComputedStyle({
```

Reply:
I would leave it as is as all other tests are written in that manner..

Reply:
I guess it depends on the lens used: `screen.getByTestId('component')` has 15 matches in the codebase, `container.firstChild`  has 276 matches.

Reply:
Refactored all tests to use `container.firstChild` 👍